src/st/dai/unused_5A69C.c

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "dai.h"
+
+STATIC_PAD_BSS(0x60);
```

Comment:
I'm not sure there's an advantage to having this in C. It's not needed for portability and doesn't appear to be referenced.